velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -478,6 +481,25 @@ TEST_F(StringTest, replace) {
       "123\u6570data");
 }
 
+TEST_F(StringTest, right) {
+  const auto right = [&](const std::optional<std::string>& str,
+                        const std::optional<int32_t>& length) {
```

Comment:
Why do we make length as optional in the test? It seems it is a required argument for right function.